2024-09-20 02:51:24 [ERROR]: Ошибка синхронизации базы данных: отношение "public.Users" не существует {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 143,
    "name": "error",
    "severity": "ОШИБКА",
    "code": "42P01",
    "file": "namespace.c",
    "line": "432",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "original": {
    "length": 143,
    "name": "error",
    "severity": "ОШИБКА",
    "code": "42P01",
    "file": "namespace.c",
    "line": "432",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));",
  "parameters": {},
  "stack": "Error\n    at Query.run (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\dialects\\postgres\\query.js:50:25)\n    at C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.createTable (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:98:12)\n    at async Function.sync (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\model.js:942:7)\n    at async Sequelize.sync (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\sequelize.js:377:9)"
}
2024-09-20 02:56:51 [ERROR]: Ошибка синхронизации базы данных: отношение "public.Users" не существует {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 143,
    "name": "error",
    "severity": "ОШИБКА",
    "code": "42P01",
    "file": "namespace.c",
    "line": "432",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "original": {
    "length": 143,
    "name": "error",
    "severity": "ОШИБКА",
    "code": "42P01",
    "file": "namespace.c",
    "line": "432",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));",
  "parameters": {},
  "stack": "Error\n    at Query.run (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\dialects\\postgres\\query.js:50:25)\n    at C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.createTable (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:98:12)\n    at async Function.sync (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\model.js:942:7)\n    at async Sequelize.sync (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\sequelize.js:377:9)"
}
2024-09-23 12:57:30 [ERROR]: Ошибка синхронизации базы данных: ������������ "your_local_db_user" �� ������ �������� ����������� (�� ������) {
  "name": "SequelizeConnectionError",
  "parent": {
    "length": 130,
    "name": "error",
    "severity": "�����",
    "code": "28P01",
    "file": "auth.c",
    "line": "326",
    "routine": "auth_failed"
  },
  "original": {
    "length": 130,
    "name": "error",
    "severity": "�����",
    "code": "28P01",
    "file": "auth.c",
    "line": "326",
    "routine": "auth_failed"
  },
  "stack": "SequelizeConnectionError: ������������ \"your_local_db_user\" �� ������ �������� ����������� (�� ������)\n    at Client._connectionCallback (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\pg\\lib\\client.js:326:19)\n    at Client._handleErrorMessage (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\pg\\lib\\client.js:346:19)\n    at Connection.emit (node:events:513:28)\n    at C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\pg\\lib\\connection.js:116:12\n    at Parser.parse (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\pg-protocol\\dist\\parser.js:36:17)\n    at Socket.<anonymous> (C:\\Users\\alexx\\Documents\\Test Squad\\employee-management-api\\node_modules\\pg-protocol\\dist\\index.js:11:42)\n    at Socket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:315:12)\n    at readableAddChunk (node:internal/streams/readable:289:9)"
}
2024-10-03 15:36:42 [ERROR]: Ошибка синхронизации базы данных: column "vacationDates" cannot be cast automatically to type timestamp with time zone[] {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 237,
    "name": "error",
    "severity": "ERROR",
    "code": "42804",
    "hint": "You might need to specify \"USING \"vacationDates\"::timestamp with time zone[]\".",
    "file": "tablecmds.c",
    "line": "11477",
    "routine": "ATPrepAlterColumnType",
    "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];"
  },
  "original": {
    "length": 237,
    "name": "error",
    "severity": "ERROR",
    "code": "42804",
    "hint": "You might need to specify \"USING \"vacationDates\"::timestamp with time zone[]\".",
    "file": "tablecmds.c",
    "line": "11477",
    "routine": "ATPrepAlterColumnType",
    "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];"
  },
  "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Function.sync (/app/node_modules/sequelize/lib/model.js:984:11)\n    at async Sequelize.sync (/app/node_modules/sequelize/lib/sequelize.js:377:9)"
}
2024-10-03 15:39:09 [ERROR]: Ошибка синхронизации базы данных: column "vacationDates" cannot be cast automatically to type timestamp with time zone[] {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 237,
    "name": "error",
    "severity": "ERROR",
    "code": "42804",
    "hint": "You might need to specify \"USING \"vacationDates\"::timestamp with time zone[]\".",
    "file": "tablecmds.c",
    "line": "11477",
    "routine": "ATPrepAlterColumnType",
    "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];"
  },
  "original": {
    "length": 237,
    "name": "error",
    "severity": "ERROR",
    "code": "42804",
    "hint": "You might need to specify \"USING \"vacationDates\"::timestamp with time zone[]\".",
    "file": "tablecmds.c",
    "line": "11477",
    "routine": "ATPrepAlterColumnType",
    "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];"
  },
  "sql": "ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP NOT NULL;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" DROP DEFAULT;ALTER TABLE \"public\".\"Users\" ALTER COLUMN \"vacationDates\" TYPE TIMESTAMP WITH TIME ZONE[];",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Function.sync (/app/node_modules/sequelize/lib/model.js:984:11)\n    at async Sequelize.sync (/app/node_modules/sequelize/lib/sequelize.js:377:9)"
}
2024-10-03 16:02:14 [ERROR]: Не удалось запустить приложение: column "hireDate" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "original": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.select (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async Function.findAll (/app/node_modules/sequelize/lib/model.js:1140:21)\n    at async Function.findOne (/app/node_modules/sequelize/lib/model.js:1240:12)\n    at async /app/app.js:71:27"
}
2024-10-03 16:04:25 [ERROR]: Не удалось запустить приложение: column "hireDate" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "original": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.select (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async Function.findAll (/app/node_modules/sequelize/lib/model.js:1140:21)\n    at async Function.findOne (/app/node_modules/sequelize/lib/model.js:1240:12)\n    at async /app/app.js:71:27"
}
2024-10-03 16:22:51 [ERROR]: Не удалось запустить приложение: column "hireDate" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "original": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.select (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async Function.findAll (/app/node_modules/sequelize/lib/model.js:1140:21)\n    at async Function.findOne (/app/node_modules/sequelize/lib/model.js:1240:12)\n    at async /app/app.js:71:27"
}
2024-10-03 16:28:56 [ERROR]: Не удалось запустить приложение: column "hireDate" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "original": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.select (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async Function.findAll (/app/node_modules/sequelize/lib/model.js:1140:21)\n    at async Function.findOne (/app/node_modules/sequelize/lib/model.js:1240:12)\n    at async /app/app.js:71:27"
}
2024-10-03 16:31:05 [ERROR]: Не удалось запустить приложение: column "hireDate" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "original": {
    "length": 170,
    "name": "error",
    "severity": "ERROR",
    "code": "42703",
    "hint": "Perhaps you meant to reference the column \"User.birthDate\".",
    "position": "366",
    "file": "parse_relation.c",
    "line": "3729",
    "routine": "errorMissingColumn",
    "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;"
  },
  "sql": "SELECT \"id\", \"firstName\", \"lastName\", \"middleName\", \"birthDate\", \"phone\", \"email\", \"programmingLanguage\", \"country\", \"bankCard\", \"password\", \"role\", \"registrationDate\", \"lastLoginDate\", \"salary\", \"lastSalaryIncreaseDate\", \"position\", \"mentorName\", \"vacationDates\", \"githubLink\", \"linkedinLink\", \"adminNote\", \"currentProject\", \"englishLevel\", \"workingHoursPerWeek\", \"hireDate\", \"createdAt\", \"updatedAt\" FROM \"public\".\"Users\" AS \"User\" WHERE \"User\".\"role\" = 'admin' LIMIT 1;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.select (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async Function.findAll (/app/node_modules/sequelize/lib/model.js:1140:21)\n    at async Function.findOne (/app/node_modules/sequelize/lib/model.js:1240:12)\n    at async /app/app.js:71:27"
}
2025-10-25 10:17:29 [ERROR]: Ошибка синхронизации базы данных: could not open relation with OID 16396 {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 97,
    "name": "error",
    "severity": "ERROR",
    "code": "XX000",
    "file": "relation.c",
    "line": "62",
    "routine": "relation_open",
    "sql": "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a, pg_namespace s WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'Users' AND s.oid = t.relnamespace AND s.nspname = 'public' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;"
  },
  "original": {
    "length": 97,
    "name": "error",
    "severity": "ERROR",
    "code": "XX000",
    "file": "relation.c",
    "line": "62",
    "routine": "relation_open",
    "sql": "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a, pg_namespace s WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'Users' AND s.oid = t.relnamespace AND s.nspname = 'public' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;"
  },
  "sql": "SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a, pg_namespace s WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'Users' AND s.oid = t.relnamespace AND s.nspname = 'public' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.showIndex (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:254:12)\n    at async Function.sync (/app/node_modules/sequelize/lib/model.js:988:29)\n    at async Sequelize.sync (/app/node_modules/sequelize/lib/sequelize.js:377:9)"
}
2025-10-25 10:17:44 [ERROR]: Ошибка синхронизации базы данных: relation "public.Users" does not exist {
  "name": "SequelizeDatabaseError",
  "parent": {
    "length": 110,
    "name": "error",
    "severity": "ERROR",
    "code": "42P01",
    "file": "namespace.c",
    "line": "429",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "original": {
    "length": 110,
    "name": "error",
    "severity": "ERROR",
    "code": "42P01",
    "file": "namespace.c",
    "line": "429",
    "routine": "RangeVarGetRelidExtended",
    "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));"
  },
  "sql": "CREATE TABLE IF NOT EXISTS \"public\".\"Notifications\" (\"id\"   SERIAL , \"message\" VARCHAR(255) NOT NULL, \"userId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"relatedUserId\" INTEGER NOT NULL REFERENCES \"public\".\"Users\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE, \"isRead\" BOOLEAN DEFAULT false, \"type\" VARCHAR(255) NOT NULL DEFAULT 'general', \"eventDate\" TIMESTAMP WITH TIME ZONE, \"createdAt\" TIMESTAMP WITH TIME ZONE NOT NULL, \"updatedAt\" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY (\"id\"));",
  "parameters": {},
  "stack": "Error\n    at Query.run (/app/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)\n    at /app/node_modules/sequelize/lib/sequelize.js:315:28\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async PostgresQueryInterface.createTable (/app/node_modules/sequelize/lib/dialects/abstract/query-interface.js:98:12)\n    at async Function.sync (/app/node_modules/sequelize/lib/model.js:942:7)\n    at async Sequelize.sync (/app/node_modules/sequelize/lib/sequelize.js:377:9)"
}
